common/Fabric-Install-Utilities.psm1

```diff
@@ -587,12 +587,16 @@ function Write-Console($message){
 
 function Test-DiscoveryHasBuildVersion($discoveryUrl, $credential) {
     $response = [xml](Invoke-RestMethod -Method Get -Uri "$discoveryUrl/`$metadata" -Credential $credential -ContentType "application/xml")
-	
+
     return $response.Edmx.DataServices.Schema.EntityType.Property.Name -contains 'BuildNumber'
 }
 
 function Add-DiscoveryRegistration($discoveryUrl, $credential, $discoveryPostBody) {
-    $hasVersion = Test-DiscoveryHasBuildVersion $discoveryUrl $credential
+    # append default /v1 if version does not exist
+    if($discoveryUrl -notmatch '/v\d$') {
```

Comment:
if the value has a trailing "/" this will appedna second /v1 correct?   so DiscoveryService/v1/  beceomes DiscoveryService/v1/v1?

Reply:
yes you're right, will update